strings/src/main/res/values/strings.xml

```diff
@@ -489,7 +489,7 @@
     <string name="offline_layer_dialog_title">Select offline layer</string>
 
     <!-- Description of how to add offline layers to the project -->
-    <string name="offline_layer_dialog_description">Select the offline layer to use for all maps in this project. You can add options to the list by selecting an MBTile file from your device.</string>
+    <string name="offline_layer_dialog_description">Select the offline layer to use for all maps in this project. You can add options to the list by selecting an MBTile file from your device and also delete existing layers from the list.</string>
```

Comment:
The file format is MBTiles with an s. While we're changing this string, could we change that as well, please? Sorry, not totally related but I think it makes sense to do in a single commit.